README.md

```diff
@@ -1,8 +1,7 @@
 # PSR-7 Storage-less HTTP Sessions
 
-[![Build Status](https://travis-ci.org/psr7-sessions/storageless.svg)](https://travis-ci.org/psr7-sessions/storageless)
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/psr7-sessions/storageless/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/psr7-sessions/storageless/?branch=master)
-[![Code Coverage](https://scrutinizer-ci.com/g/psr7-sessions/storageless/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/psr7-sessions/storageless/?branch=master)
+[![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fpsr7-sessions%2Fstorageless)](https://dashboard.stryker-mutator.io/reports/github.com/psr7-sessions/storageless/master)
```

Comment:
Note: this badge does not work yet. I need someone else to head to https://dashboard.stryker-mutator.io/ and configure the API keys for it (https://infection.github.io/guide/mutation-badge.html#main) in the repository secrets (https://github.com/psr7-sessions/storageless/settings/secrets).

The two environment variables are called `INFECTION_BADGE_API_KEY` and `STRYKER_DASHBOARD_API_KEY`.

Reply:
Moved to #135